### PR TITLE
Use a cryptographically secure RNG to generate admin passwords.

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

File: azure-devops/create-vmss.ps1 [hunk 1]
```diff
@@ -86,18 +86,36 @@ Generates a random password.
...
+only alphanumeric characters, underscore, and dash
 
 .PARAMETER Length
 The length of the returned password.
 #>
 function New-Password {
   Param ([int] $Length = 32)
 
-  $Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+  # 6 bits
```
Comment (CaseyCarter) [inline azure-devops/create-vmss.ps1:97]: This comment is a bit inscrutable. I think you mean something like "$chars is a 64 character alphabet, so each randomly-selected member we append to our password adds log_2 64 = 6 bits of entropy".

File: azure-devops/create-vmss.ps1 [hunk 2]
```diff
@@ -86,18 +86,36 @@ Generates a random password.
 
 .DESCRIPTION
 New-Password generates a password, randomly, of length $Length, containing
-only alphanumeric characters (both uppercase and lowercase).
+only alphanumeric characters, underscore, and dash
```
Comment (StephanTLavavej) [inline azure-devops/create-vmss.ps1:89]:
```suggestion
only alphanumeric characters, underscore, and dash.
```

File: azure-devops/create-vmss.ps1 (hunk 1 above)
Comment (StephanTLavavej) [reply azure-devops/create-vmss.ps1:97]: The power-of-2 length is also important because of the bitwise-AND with `$mask`. I can push a simple comment.

**Comment by StephanTLavavej** [comment]
Thanks for improving this VM setup script!

**Pull request merged**